Scenarios/Counters/PutCommentsTest.cs

```
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Linq;
+using TestingEnvironment.Client;
+
+namespace Counters
+{
+    public class PutCommentsTest : BaseTest
+    {
+        public PutCommentsTest(string orchestratorUrl, string testName) : base(orchestratorUrl, testName, "Aviv")
+        {
+        }
+
+        public override void RunActualTest()
+        {
+            using (DocumentStore.Initialize())
+            {
+                using (var session = DocumentStore.OpenSession())
+                {
+                    var count = session.Query<BlogComment>().Count();
+                    if (count >= 10 * 1024)
+                    {
+                        ReportInfo("Aborting BlogComment documents insertion, we already have enough docs");
+                        return;
+                    }
+                }
+
+                ReportInfo("Inserting BlogComment documents");
+
+                var numOfRetries = 3;
+                while (true)
+                {
+                    try
+                    {
+                        PutBlogCommentDocs();
+                    }
+                    catch (Exception e)
+                    {
+                        if (--numOfRetries > 0)

```

Let's report the errors as well

---

What kind of issues are we expecting here?